src/genie/libs/parser/iosxe/tests/ShowLispEidTableVrfUserIpv4MapCache/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,571 @@
+expected_output = {
+    "vrf": "User",
+    "iid": 4100,
+    "number_of_entries": 2186,
+    "eid": {
+        "0.0.0.0/0": {
+            "subnet": "/0",
+            "uptime": "1w6d",
+            "expire": "never",
+            "source": [
+                "static-send-map-request"
+            ],
+            "rloc": {
+                "status": "negative_cache_entry",
+                "action": "send-map-request"
+            }
+        },
+        "0.0.0.0": {
+            "subnet": "/5",
+            "uptime": "4w6d",
+            "expire": "00:14:38",
+            "source": [
+                "map-reply",
+                "forward-native"
+            ],
+            "rloc": {
+                "status": "encapsulating_to_proxy_etr"
+            }
+        },
+        "8.0.0.0": {
+            "subnet": "/7",
+            "uptime": "07:18:55",
+            "expire": "00:07:19",
+            "source": [
+                "map-reply",
+                "forward-native"
+            ],
+            "rloc": {
+                "status": "encapsulating_to_proxy_etr"
+            }
+        },
+        "10.0.0.0": {
+            "subnet": "/9",
+            "uptime": "1w4d",
+            "expire": "00:00:13",
+            "source": [
+                "map-reply",
+                "forward-native"
+            ],
+            "rloc": {
+                "status": "encapsulating_to_proxy_etr"
+            }
+        },
+        "10.128.0.0": {
+            "subnet": "/11",
+            "uptime": "4w6d",
+            "expire": "00:04:45",
+            "source": [
+                "map-reply",
+                "forward-native"
+            ],
+            "rloc": {
+                "status": "encapsulating_to_proxy_etr"
+            }
+        },
+        "10.16.0.17": {
+            "subnet": "/32",
+            "uptime": "1w6d",
+            "expire": "13:36:24",
+            "source": [
+                "map-reply",
+                "self",
+                "complete"
+            ],
+            "rloc": {
+                "ip": "10.8.129.94",
+                "uptime": "1w6d",
+                "state": "up",
+                "priority": 10,
+                "weight": 10,
+                "encap_iid": ""
```

Comment:
take the same with output as is.
```suggestion
                "encap_iid": "-"
```

Reply:
Updated